### PR TITLE
Add REFRESH MATERIALIZED VIEW syntax

### PR DESCRIPTION
**Pull request by joy-yao**

### CODE REVIEW

File: presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4 [hunk 1]
```diff
@@ -440,6 +441,8 @@ CREATE: 'CREATE';
 TABLE: 'TABLE';
 VIEW: 'VIEW';
 REPLACE: 'REPLACE';
+REFRESH: 'REFRESH';
```
Comment (electrum) [inline presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4]: These should be non-reserved

File: presto-parser/src/main/java/com/facebook/presto/sql/tree/AstVisitor.java [hunk 2]
```diff
@@ -457,6 +457,11 @@ protected R visitCreateView(CreateView node, C context)
         return visitStatement(node, context);
     }
 
+    protected R visitRefreshMaterializedView(RefreshMaterializedView node, C context)
```
Comment (electrum) [inline presto-parser/src/main/java/com/facebook/presto/sql/tree/AstVisitor.java]: Move this so it's not between create and drop view

**Comment by martint** [comment]
Before we add any language support, I'd like to understand how this is going to work. There's no notion of materialized views in the Presto engine, and if it's going to be a connector-specific concept, we may want to model this as callable functions instead of having dedicated syntax.

FWIW, this is not standard syntax, either.

In the long term, I'd like materialization of a view to be an optimization based on cpu/space tradeoffs and knowledge of dependencies and latency requirements (e.g., a query over a view needs to finish before a given time, so the engine chooses to materialize portions to satisfy that requirement).

**Comment by electrum** [comment]
We could do this for now as stored procedure calls.

**Comment by martint** [comment]
> We could do this for now as stored procedure calls.

:+1:

**Comment by joy-yao** [comment]
Closing. Opened https://github.com/prestodb/presto/pull/5237 instead.

**Pull request closed without merging**